portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data; 
+
+public final class Comment {
+
+    private final long id;
+    private final String name;
+    private final long timestamp;
```

Comment:
I'm wondering if a Timestamp or Date type might make more sense for the timespamp (e.g. https://cloud.google.com/datastore/docs/concepts/entities#properties_and_value_types). What do you think?

Reply:
I changed it so that it converts the long (the milliseconds) into a Date object in the javascript code - this makes more sense because I can print it in different formats. Thanks for the suggestion!